interfaces/builtin/x11.go

```diff
@@ -19,19 +19,96 @@
 
 package builtin
 
-const x11Summary = `allows interacting with the X11 server`
+import (
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/interfaces/apparmor"
+	"github.com/snapcore/snapd/interfaces/seccomp"
+	"github.com/snapcore/snapd/interfaces/udev"
+	"github.com/snapcore/snapd/release"
+	"github.com/snapcore/snapd/snap"
+
+	"strings"
+)
+
+const x11Summary = `allows interacting with an X11 server`
```

Comment:
`Allows interacting with or running as an X11 server` (since the slot now comes around)

Reply:
Done